ReleaseProcessScript/tools/PesterTester/semver_functions.Tests.ps1

```diff
@@ -27,21 +27,21 @@ Describe "semver_functions" {
 
     It "Get-Possible-Next-Versions-Develop_WithRc_ShouldReturnArray" {
       $Version = "1.2.0-rc.4"
-      $NextVersions = "1.2.0-rc.5", "1.2.0", "2.0.0-alpha.1", "2.0.0-beta.1", "2.0.0"
+      $NextVersions = "1.2.0", "2.0.0-alpha.1", "2.0.0-beta.1", "2.0.0"
 
       Get-Possible-Next-Versions-Develop $Version | Should Be $NextVersions
     }
 
     It "Get-Possible-Next-Versions-Develop_WithoutPre_ShouldReturnArray" {
       $Version = "1.2.0"
-      $NextVersions = "1.3.0-alpha.1", "1.3.0-beta.1", "1.3.0", "2.0.0-alpha.1", "2.0.0-beta.1", "2.0.0"
+      $NextVersions = "1.3.0-alpha.1", "1.3.0-beta.1", "1.3.0", "2.0.0", "2.0.0-alpha.1", "2.0.0-beta.1"
```

Comment:
please explain new ordering

Reply:
I can't remember too well, I believe it's just a leftover of experimentation, since the test is green after reverting. I'll just revert this change, then

Reply:
I think I know why we changed the ordering. The production code is like this:
```
return $NextPossiblePatch, "$($NextPossiblePatch)-alpha.1", "$($NextPossiblePatch)-beta.1"
```
(It was like that before, we didn't touch that part). I believe we changed the order to reflect this code bit, but I can nonetheless revert the changes, since it doesn't affect the tests

Reply:
I see. I assumed that the order of $NextVersions was included in the assertion. Obviously it is not. In this instance it might be a good idea though? Do we have a place where we assert that the "correct" order of items is printed on the next-version prompt? Aside from that, it's okay to leave the order in the assertion in the updated form.

Also, just in case this is the spot where the list of next-versions is checked. Okay, first we go up and then we offer the 2.0-prerelease versions only as a last resort. this kind of makes sense. So, new list is fine.